pkg/descheduler/descheduler.go

```diff
@@ -109,11 +109,6 @@ func RunDeschedulerStrategies(ctx context.Context, rs *options.DeschedulerServer
 		ignorePvcPods = *deschedulerPolicy.IgnorePVCPods
 	}
 
-	maxNoOfPodsToEvictPerNode := rs.MaxNoOfPodsToEvictPerNode
-	if deschedulerPolicy.MaxNoOfPodsToEvictPerNode != nil {
-		maxNoOfPodsToEvictPerNode = *deschedulerPolicy.MaxNoOfPodsToEvictPerNode
-	}
-
```

Comment:
shouldn't we keep this nil check instead of passing the variable directly?

Reply:
I check for the nil inside the pod evictor

Reply:
I noticed that after I commented here, sorry. Still, is there any reason not to do that check earlier with the rest of the flags that are checked similarly here?

Reply:
Nil is a valid value

Reply:
Got it, the other thread cleared that up for me too thanks